libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/NavigationCamera.kt

```diff
@@ -282,6 +284,7 @@ class NavigationCamera(
         frameTransitionOptions: NavigationCameraTransitionOptions = DEFAULT_FRAME_TRANSITION_OPT,
         transitionEndListener: TransitionEndListener? = null,
     ) {
+        isFromNavigationCamera = true
```

Comment:
@LukasPaczos 
I am setting this boolean flag to true inside these 2 methods `requestCameraModeToFollowing` and `requestCameraModeToOverview` which makes sense because these functions can only be invoked on `NavigationCamera`. So if the use case were as follows:
- User invokes `requestCameraModeToFollowing` followed by immediately
- User invokes `requestCameraModeToOverview`, the NavigationCamera will not go to idle state because this flag is set to true

However, if the use case is
- User invokes `requestCameraModeToFollowing` followed by immediately
- invoking an operation on mapView camera, I somehow need to set this `isFromNAvigationCamera` flag to `false`.

This is what I am trying to figure out as to how to do that and what would be the correct place to set it to false. Wdyt?

Reply:
Okay so I think I figured it out on how to set it to false. The answer is in `NavigationScaleGestureHandler` and as per public documentation 
```
### Interruptions
 * Any other camera change request outside of the [NAVIGATION_CAMERA_OWNER] and scale
 * gestures ([MapAnimationOwnerRegistry.GESTURES]) will immediately make [NavigationCamera] fallback
 * to [NavigationCameraState.IDLE] state to avoid running competing animations.
```
I am hooking onto the `owner` of the transition request. Based on the owner I decide whether to set this to true or false.